### DISCUSSION TITLE
Boolean union duplicates geometry

### DISCUSSION THREAD

**Discussion by khorev001** [clicketyclackety/Crash#39] [General]
Hi, thanx for an outstanding addon!

While testing we found a bug. When i was trying to use BooleanUnion command in rhino all the geometry got duplicated and other person was not able to see new geometry that was produced by union.

**Comment by clicky** [comment]
Hey @khorev001, thanks for using Crash!
I've been told this now by a few users. Once I merge in #40 I will create a fix for this as I'm sure this is quite irritating 😊
